sympy/matrices/benchmarks/bench_matrix.py

```diff
@@ -1,10 +1,11 @@
 from __future__ import print_function, division
```

Comment:
I didn't know this file existed, but it'd be nice if all of this was in our benchmark suite:

https://github.com/sympy/sympy_benchmarks